### PR TITLE
Bug 1159885 - Status bar is not visible on multiple attention windows

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3d818c5dbde6b10fdb6b24c642e46344a7f78b6a

File: apps/system/js/statusbar.js [hunk 1]
```diff
@@ -468,7 +470,7 @@ var StatusBar = {
   },
 
   _updateIconVisibility: function sb_updateIconVisibility() {
-    if (this.isPaused()) {
+    if (this._paused !== 0) {
```
Comment (etiennesegonzac) [inline apps/system/js/statusbar.js]: why this change?

**Comment by etiennesegonzac** [comment]
can you change just the `attentionopened` test to use a
`window.dispatchEvent('attentionopened')` instead of the `handleEvent` call to cover the missing event listener you added?

File: apps/system/js/statusbar.js (hunk 1 above)
Comment (gmarty) [inline apps/system/js/statusbar.js]: Ooopsie!

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=029a2c2c25e577af95931f83797653166c1230d8

**Comment by gmarty** [comment]
There's no test for `attentionopened`, so I added one!

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1612c20962ab190e75302fd18f752fb456d180ae

**Comment by mikehenrty** [comment]
closed in favor of #29835
